articles/azure-app-configuration/faq.yml

```diff
@@ -146,15 +141,16 @@ sections:
 
       - question: Why can't I create an App Configuration store with the same name as one that I just deleted?
```

Comment:
it should be ".......as the one that I just deleted?" 

I definitely have spent too much time on docs in the last two weeks :D

Reply:
Yes, it missed the "the". This is not a new Q added by this PR. Changing it will break any saved links. We can change questions if it's significant. I will leave this one as is. This reminded us that we need to be very careful when we add a new Q in the future.